### PR TITLE
Deprecate lsf server

### LINKED ISSUE #7439: Deprecate LSF_SERVER queue_option

### PR DESCRIPTION
**Pull request by jonathan-eq**
**Issue**
Resolves #7439 
Depends on #7428 

**Approach**
_Short description of the approach_

(Screenshot of new behavior in GUI if applicable)

- [x] PR title captures the intent of the changes, and is fitting for release notes.
- [x] Added appropriate release note label
- [x] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).
- [ ] Make sure tests pass locally (after every commit!)

## When applicable
- [ ] **When there are user facing changes**: Updated documentation
- [ ] **New behavior or changes to existing untested code**: Ensured that unit tests are added (See [Ground Rules](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md#ground-rules)).
- [ ] **Large PR**: Prepare changes in small commits for more convenient review
- [ ] **Bug fix**: Add regression test for the bug
- [ ] **Bug fix**: Create Backport PR to latest release

### CODE REVIEW

**Comment by xjules** [comment]
It's a bit weird that this depends on https://github.com/equinor/ert/pull/7428 and in that one @berland suggests to wait for this one until it hits stable :thinking:

**Comment by jonathan-eq** [comment]
> It's a bit weird that this depends on #7428 and in that one @berland suggests to wait for this one until it hits stable 🤔

Yes, this one if blocked. What can be merged is transferred to #7428.

**Review by xjules — APPROVED** [review]
Let's deprecate!

File: src/ert/config/parsing/config_schema_deprecations.py [hunk 1]
```diff
@@ -209,4 +209,10 @@
         "It has been used in the past to supply options to the qstat command.",
         check=lambda line: "QSTAT_OPTIONS" in line,
     ),
+    DeprecationInfo(
+        keyword="QUEUE_OPTION",
+        message="LSF_SERVER as QUEUE_OPTION to the LSF system will be removed in "
+        "the future, and it is not recommended to use this QUEUE_OPTION.",
```
Comment (berland) [inline src/ert/config/parsing/config_schema_deprecations.py:215]: I think we should sharpen this text a bit now:

"LSF_SERVER as QUEUE_OPTION is not needed and will be removed in the future. Please remove the configuration line."

**Pull request merged**